packages/runtime/test/integration/pooled-op-cost.ts

```diff
@@ -50,29 +50,41 @@ describe("TEALv5: Pooled Opcode Cost calculation", function () {
 			RUNTIME_ERRORS.TEAL.MAX_COST_EXCEEDED
 		);
 
-		// exceed even with 3 "normal transactions"
+		// exceed even with 3 "normal transactions", add note to make txn different with each other
 		expectRuntimeError(
 			() =>
 				runtime.executeTx([
 					appCallParam,
-					{ ...appCallParam, appArgs: [`str:${STR_NORMAL_COST}`] },
-					{ ...appCallParam, appArgs: [`str:${STR_NORMAL_COST}`] },
-					{ ...appCallParam, appArgs: [`str:${STR_NORMAL_COST}`] },
+					{
+						...appCallParam,
+						appArgs: [`str:${STR_NORMAL_COST}`],
```

Comment:
This line is same in all objects in this array. Maybe we can include that itself in `appCallParam`

Reply:
No, it's different object. I overwrite payFlags and appCallParam also use in other test, so we should not change it.